cli/ioctl/util/util.go

```diff
@@ -24,3 +34,57 @@ func ConnectToEndpoint() (*grpc.ClientConn, error) {
 	}
 	return grpc.Dial(endpoint, grpc.WithInsecure())
 }
+
+// IotxStringToRau convert Iotx string into Rau big int
+func IotxStringToRau(amount string) (*big.Int, error) {
+	amountStrings := strings.Split(amount, ".")
+	if len(amountStrings) == 1 {
+		for i := 0; i < IotxDecimalNum; i++ {
+			amountStrings[0] += "0"
+		}
+		amountRau, ok := big.NewInt(0).SetString(amountStrings[0], 10)
+		if !ok {
+			return nil, fmt.Errorf("failed to convert string into big int")
+		}
+		return amountRau, nil
+	}
+	if len(amountStrings) > 2 || len(amountStrings[1]) > IotxDecimalNum {
```

Comment:
Is it better to use https://golang.org/pkg/regexp/ to guarantee the decimal string pattern.